test/e2e/src/pom/highlightsGraphPage.ts

```diff
@@ -0,0 +1,14 @@
+import { Page } from '@playwright/test';
+import { VideoComponent } from '../../components/videoComponent';
+const HIGHLIGHTS_GRAPH_PAGE_VIDEO_SELECTOR = '//*[@id="player_html5_api"]';
+
+/**
+ * Video player examples highlight graph page object
+ */
+export class HighlightsGraphPage {
```

Comment:
I would consult with @refael-m  about this one, but i think that having to create a pom for each page might not be scaleable, maybe there is a generic pom page class factory and an instance is created on the fly with each test ...since essentially all these pages are going to have a video player and you are going to validate functionality on that video player so that generic class factory can be shared across all pages